src/types/asset.ts

```diff
@@ -24,6 +24,7 @@ export interface Token extends ERC20 {
   priceUsdc: Usdc;
   supported: {
     zapper?: boolean;
+    wftm?: boolean;
```

Comment:
Here we need to have a naming convention that is synced with yearn-meta. Maybe `wftmZap` just to be more explicit? Contract is named `ftmApeZap`, we can also use that.

Reply:
renamed to `ftmApeZap`